core/includes/bootstrap.inc

```diff
@@ -1530,10 +1534,10 @@ function backdrop_page_get_cache($check_only = FALSE, $return_stale = FALSE) {
 /**
  * Determines the cacheability of the current page.
  *
- * @param $allow_caching
+ * @param bool|null $allow_caching
  *   Set to FALSE if you want to prevent this page from being cached.
```

Comment:
It can be FALSE as well - as the comment states. And that's how it's actually used in core.